code/modules/surgery/organs/internal/ears/_ears.dm

```diff
@@ -159,10 +157,26 @@
 		return FALSE
 	return TRUE
 
-/datum/bodypart_overlay/mutant/cat_ears/color_image(image/overlay, draw_layer, obj/item/bodypart/limb)
-	if(draw_layer != bitflag_to_layer(colorless_layer))
-		return ..()
-	return overlay
+/datum/bodypart_overlay/mutant/cat_ears/get_image(image_layer, obj/item/bodypart/limb)
+	var/mutable_appearance/base_ears = ..()
+
+	// Construct image of inner ears, apply to base ears as an overlay
+	var/gender = (limb?.limb_gender == FEMALE) ? "f" : "m"
+	var/list/icon_state_builder = list()
+	icon_state_builder += sprite_datum.gender_specific ? gender : "m"
+	icon_state_builder += "[feature_key]inner"
+	icon_state_builder += get_base_icon_state()
+	icon_state_builder += mutant_bodyparts_layertext(image_layer)
+
+	var/finished_icon_state = icon_state_builder.Join("_")
+
+	var/mutable_appearance/inner_ears = mutable_appearance(sprite_datum.icon, finished_icon_state, layer = image_layer, appearance_flags = RESET_COLOR)
+
+	if(sprite_datum.center)
+		center_image(inner_ears, sprite_datum.dimension_x, sprite_datum.dimension_y)
```

Comment:
This is just a copypaste of the parent proc. The image construction for the mutant external organs is somewhat sensitive and I prefer not to have it spread around multiple places. Solution below is kinda hacky but for most cases prevents people having to debug multiple icon generations at once

```suggestion
/datum/bodypart_overlay/mutant/cat_ears/get_image(image_layer, obj/item/bodypart/limb)
	var/mutable_appearance/base_ears = ..()

	// Construct image of inner ears, apply to base ears as an overlay
	feature_key += "inner"
	var/mutable_appearance/inner_ears = ..()
	inner_ears.appearance_flags = RESET_COLOR
	feature_key = initial(feature_key)
	
```
You could also section it further into another proc but these are already a little hot and I don't think too much is gained by it

Reply:
hrmn, I dislike it for being hacky- but hey so was my solution, and yeah I feel the logic for doing it this way's sound